markhor_control/config/wheels.yaml

```diff
@@ -1,43 +1,44 @@
-  diff_drive_controller:
-    type: "diff_drive_controller/DiffDriveController"
-    left_wheel: ['front_left_wheel', 'rear_left_wheel']
-    right_wheel: ['front_right_wheel', 'rear_right_wheel']
-    publish_rate: 50
-    pose_covariance_diagonal: [0.001, 0.001, 0.001, 0.001, 0.001, 0.03]
-    twist_covariance_diagonal: [0.001, 0.001, 0.001, 0.001, 0.001, 0.03]
-    cmd_vel_timeout: 0.25
-    velocity_rolling_window_size: 2
+  tracks:
+    # Diff drive controller
+    diff_drive_controller:
+      type: "diff_drive_controller/DiffDriveController"
+      left_wheel: ['front_left_wheel', 'rear_left_wheel']
+      right_wheel: ['front_right_wheel', 'rear_right_wheel']
+      publish_rate: 50
+      pose_covariance_diagonal: [0.001, 0.001, 0.001, 0.001, 0.001, 0.03]
+      twist_covariance_diagonal: [0.001, 0.001, 0.001, 0.001, 0.001, 0.03]
+      cmd_vel_timeout: 0.25
+      velocity_rolling_window_size: 2
 
-    wheel_separation : 1
-    wheel_radius : 1
+      wheel_separation : 1
+      wheel_radius : 1
 
-    # Base frame_id
-    base_frame_id: markhor_root
+      # Base frame_id
+      base_frame_id: markhor_root
 
-    # Odometry fused with IMU is published by robot_localization, so
-    # no need to publish a TF based on encoders alone.
-    enable_odom_tf: false
+      # Odometry fused with IMU is published by robot_localization, so
+      # no need to publish a TF based on encoders alone.
+      enable_odom_tf: false
 
-    # Hardware provides wheel velocities
-    estimate_velocity_from_position: false
+      # Hardware provides wheel velocities
+      estimate_velocity_from_position: false
 
-    # Wheel separation and radius multipliers
-    wheel_separation_multiplier: 1.0 # default: 1.0
-    wheel_radius_multiplier    : 1.0 # default: 1.0
+      # Wheel separation and radius multipliers
+      wheel_separation_multiplier: 1.0 # default: 1.0
+      wheel_radius_multiplier    : 1.0 # default: 1.0
 
-    # Velocity and acceleration limits
-    # Whenever a min_* is unspecified, default to -max_*
-    linear:
-      x:
-        has_velocity_limits    : true
-        max_velocity           : 1.0   # m/s
-        has_acceleration_limits: true
-        max_acceleration       : 3.0   # m/s^2
-    angular:
-      z:
-        has_velocity_limits    : true
-        max_velocity           : 2.0   # rad/s
-        has_acceleration_limits: true
-        max_acceleration       : 6.0   # rad/s^2
\ No newline at end of file
+      # Velocity and acceleration limits
+      # Whenever a min_* is unspecified, default to -max_*
+      linear:
+        x:
+          has_velocity_limits    : true
+          max_velocity           : 1.0   # m/s
+          has_acceleration_limits: true
+          max_acceleration       : 3.0   # m/s^2
+      angular:
+        z:
+          has_velocity_limits    : true
+          max_velocity           : 2.0   # rad/s
+          has_acceleration_limits: true
+          max_acceleration       : 6.0   # rad/s^2
```

Comment:
no changes... issue avec les line endings

Reply:
Ehh no, this is necessary so that the diff_drive controller is in the same separated namespace for the tracks. 

And what do you mean by issues with the line endings ? Do you mean that there's no line ending at the end of the file ?

Reply:
Est-ce qu'il y a un changement?

Dans le diff de github sa me dis que toutes les lignes ont ete modifiées, je pensais que c'etais une histoire de line endings linux vs windows

Reply:
No, it's because of the indentation. At line #2, there's a new level that as been added to add a separated namespace for the tacks. The reason all the line were change was that the new level added required a new level of indentation, that's the more green part in front of each line.